common/journal_entries/imperia_qing_journal_entries.txt

```diff
@@ -405,7 +363,7 @@ je_establish_local_armies = {
 			limit = {
 				is_subject_of = c:CHI
 			}
-			c:CHI = {
+			c:CHI ?= {
```

Comment:
This is too late, you have a c:CHI in the limit. you just need to check if c:CHI exists before the first mention of it.